iphone/Classes/PlatformModule.m

```diff
@@ -227,6 +231,23 @@ -(NSString*)macaddress
     return [TiUtils appIdentifier];
 }
 
+- (NSString *)identifierForVendor
+{
+    return [[[UIDevice currentDevice] identifierForVendor] UUIDString];
+}
+
+#if defined(USE_TI_PLATFORMIDENTIFIERFORADVERTISING) || defined(USE_TI_PLATFORMGETIDENTIFIERFORADVERTISING)
+- (NSNumber *)isAdvertisingTrackingEnabled
```

Comment:
Make "isAdvertisingTrackingEnabled" property in .h file similar to 
+@property(readonly,nonatomic) NSString *identifierForAdvertising;